dev-requirements.txt

```diff
@@ -7,17 +7,17 @@ ruff==0.4.2
 import-linter~=2.0
 
 # testing
-pytest~=7.4
-pytest-asyncio~=0.18.0
-pytest-alembic~=0.9.1
+pytest~=8.2
+pytest-asyncio~=0.23.7
+pytest-alembic~=0.11.0
 pytest-httpserver~=1.0
 pytest-timeout~=2.0
 aioresponses~=0.7
 requests-mock~=1.8
 httpx~=0.24.0
```

Comment:
I could not bump the version of `httpx` to `~=0.27.0` due too `igz-mgmt`:
```
ERROR: Cannot install -r dockerfiles/mlrun-api/requirements.txt (line 7), httpx~=0.27 and igz-mgmt==0.1.3 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested httpx~=0.27
    igz-mgmt 0.1.3 depends on httpx~=0.24.1
    The user requested httpx~=0.27
    igz-mgmt 0.1.4 depends on httpx~=0.24.1

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
```

Reply:
@moranbental  fyi

Reply:
I will bump the httpx version in the igz-mgmt

Reply:
This requires dropping support for Python 3.7 in the igz-mgmt

Reply:
I think that if you loosen the specification to allow every `0.x.y` version (`httpx~=0.24` in igz-mgmt) it will support both Python 3.7 and newer httpx versions in mlrun

Reply:
please change the `igz-mgmt` version to `0.2.0`

Reply:
Thanks, will do.